WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1643,23 +1643,12 @@ - (void)logSavePostStats
     }
     
     if ([buttonTitle isEqualToString:NSLocalizedString(@"Post", nil)]) {
-        [WPAnalytics track:WPAnalyticsStatEditorPublishedPost withProperties:properties];
-        
-        if ([self.post hasPhoto]) {
-            [WPAnalytics track:WPAnalyticsStatPublishedPostWithPhoto];
-        }
-        
-        if ([self.post hasVideo]) {
-            [WPAnalytics track:WPAnalyticsStatPublishedPostWithVideo];
-        }
+        properties[WPAnalyticsStatEditorPublishedPostPropertyCategory] = @([self.post hasCategories]);
+        properties[WPAnalyticsStatEditorPublishedPostPropertyPhoto] = @([self.post hasPhoto]);
```

Comment:
Might be out of scope for this PR, but `[self.post hasPhoto]` is always `0` in my debugger even after adding images/featured image.